src/hotspot/share/opto/loopnode.cpp

```diff
@@ -3111,6 +3213,8 @@ void OuterStripMinedLoopNode::adjust_strip_mined_loop(PhaseIterGVN* igvn) {
     }
   }
 
+  handle_sunk_stores_when_finishing_construction(igvn);
```

Comment:
Above, where you insert the phis, you may want to say something about the case of Sunk Stores as well.